client/my-sites/marketing/index.js

```diff
@@ -32,6 +33,8 @@ export default function() {
 
 	paths.forEach( path => page( path, ...[ siteSelection, sites, makeLayout, clientRender ] ) );
 
+	page( '/marketing/disconnected/:service', redirectDefaultConnectionsDomain );
```

Comment:
Do you think this needs to be called `disconnected`, or could it be used for other things?

Reply:
Yes, I was thinking that this could be generally useful, so perhaps `disconnected` is a bad name. We could have it as `/marketing/connections/:service` as that's sort of what it is? The detailed view of a particular service.

Reply:
I rebased this as there was a conflict with some global file reformatting that happened yesterday. Also changed the route from `/marketing/disconnected/:service` to `/marketing/connection/:service` and this seems to test fine for me locally

Reply:
Just going to /marketing/tools/connection shows an unexpected error - don't know much about Calypso routes, should it just do a graceful redirect back to /connections instead if no service on end of path?

Reply:
We could put that redirect in, but I think the base route is `/marketing/connections`, so we should probably redirect `/marketing/connection` rather than with `tools`. Alternatively, we could leave the `connections` as plural?

Reply:
I tried leaving `connections` as plural but it broke the main page, seems to clash with the main route sometime, but if you could get that working that would make sense - but not critical, I think we would merge this with the `/marketing/connection` throwing an error an come back to it if we wanted - in theory we should never be using the route without a service param anayway.